### ISSUE TITLE
Dice Sample

### ISSUE THREAD

**Issue by BunteDimensionen** [foundryvtt/unfulfilled-rolls#23]
A few days ago I got my first GoDice and tested it right away in Foundry. Everything works except for a /r 1d20+1d20+1d20 test. The GoDice module recognizes the dice correctly, but Foundry does not display them correctly

**Comment by Hijah82** [comment]
Hey, same issue here. Do you use them with the DSA5 (The Dark Eye) System from Ulisses? I try to roll 3d20 rolls, and the Foundry GoDice Screen shows the right results, but in the Chat Window other results were printed.

**Comment by BunteDimensionen** [comment]
Yes, exactly have the problem with DSA5. The developer of DSA5 and the people of GoDice see the problem with the module "Unfulfilled-rolls" which does not implement the dice test 1W20+1W20+1W20 correctly.

Von: Hijah82 ***@***.***>
Gesendet: Dienstag, 11. Juli 2023 00:47
An: foundryvtt/unfulfilled-rolls ***@***.***>
Cc: BunteDimensionen ***@***.***>; Author ***@***.***>
Betreff: Re: [foundryvtt/unfulfilled-rolls] Dice Sample (Issue #23)


Hey, same issue here. Do you use them with the DSA5 (The Dark Eye) System from Ulisses? I try to roll 3d20 rolls, and the Foundry GoDice Screen shows the right results, but in the Chat Window other results were printed.

-
